marketplace/netlify-build/src/styles.js

```diff
@@ -0,0 +1,32 @@
+import { css } from 'emotion';
+import tokens from '@contentful/forma-36-tokens';
+
+export default {
+  info: css({
+    color: tokens.colorTextLight,
+    marginBottom: tokens.spacingM,
+    fontSize: tokens.fontSizeS,
+    fontWeight: tokens.fontWeightNormal
+  }),
+  button: css({
+    marginBottom: tokens.spacingS
+  }),
+  header: css({
+    display: 'flex',
+    marginBottom: tokens.spacingS
+  }),
+  alphaLabel: css({
+    display: 'block',
+    marginRight: tokens.spacingS,
+    background: tokens.colorBlueDark,
+    color: tokens.colorWhite,
+    padding: tokens.spacing2Xs,
+    fontSize: '0.65rem',
+    lineHeight: '0.65rem',
+    borderRadius: '3px',
+    textTransform: 'uppercase'
+  }),
+  body: css({
+    fontFamily: 'Avenir Next W01'
```

Comment:
@suevalov do you happen to know what's the current consensus on fonts we should use in extensions?

Reply:
@jelz we're going to proceed with `system` font in the web app. As far as I know the swap is planned for Q3.

Reply:
Should I leave this as-is for now or remove it?

Reply:
I'd leave `Avenir` for now and replace fonts in all extension when a swap happens